CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ else()
     add_compile_options($<$<COMPILE_LANGUAGE:CXX>:-Wsign-conversion>)
     add_compile_options($<$<COMPILE_LANGUAGE:CXX>:-Wsign-compare>)
     add_compile_options($<$<COMPILE_LANGUAGE:CXX>:-fno-rtti>)
+    add_compile_options($<$<COMPILE_LANG_AND_ID:CXX,Clang>:-fsized-deallocation>)
```

Comment:
What errors did you see without this option?

Reply:
Clang does not support sized deallocation unless -fsized-deallocation is specified. This results in the following error:

~~~
tests/move_only_function/test_unique.cpp:48:9: error: no matching function for call to 'operator delete'
        ::operator delete(s, full_size);
        ^~~~~~~~~~~~~~~~~
~~~

Either the flag must be specified, or the relevant lines must be wrapped in `#ifdef __cpp_sized_deallocation`.

Reply:
Because this is in tests, I'm going to enable the compiler option instead.